bindings_wasm/src/mls_client.rs

```diff
@@ -172,7 +173,8 @@ impl WasmClient {
     signature_bytes: Uint8Array,
     chain_id: u64,
     account_address: String,
-    chain_rpc_url: String,
+    // TODO:nm Remove this
```

Comment:
```suggestion
```

Reply:
not sure if they're removed in the next PR

Reply:
I guess I could remove this one now since it has no users. The rest I'm waiting until we are _sure_ about the required inputs, so I don't change the public API twice for no good reason.